### ISSUE TITLE
"Select project" selector undiscoverable

### ISSUE THREAD

**Issue by adriennes** [HumanCellAtlas/dcp-feedback#14]
The selector under Project name in the Data Browser is a useful feature, but was undiscoverable by participants.  The design needs to make this option more visible

**Comment by gabsie** [comment]
I don't understand the functionality of the Select links at the moment. 
Specifically in the screenshot. If I select the top project, I get the bottom one selected as well. 
(It's another problem that we have the same project name for those, will raise the dataset idea once more)
Let's think more about what we are trying to make people accomplish here, and then we think about the design solution.

![screen shot 2018-12-13 at 11 04 40](https://user-images.githubusercontent.com/19713125/49934834-5a326800-fec7-11e8-9d4f-7dd6bf31a6ea.png)

**Comment by theathorn** [comment]
@gabsie I only ever get the single project I select with the "+ Select Project" hover over. Can you still reproduce this issue?

@adriennes The "+ Select Project" hover over appears when the cursor is anywhere within the "Project Name" text and seems fairly easily discoverable. Maybe this has been improved recently? Or do you have an alternative suggestion for the UX?

**Comment by adriennes** [comment]
@theathorn - no users were able to find this feature in testing, despite many complaints about having to use the search facets to narrow down the list when they only want a single project.  Most users have requested checkboxes, which is a more familiar interaction.  Is there any reason we can't do checkboxes instead?

**Comment by theathorn** [comment]
@NoopDog @MillenniumFalconMechanic @frano-m : See feedback above r.e. using checkboxes instead of hover over.

**Comment by NoopDog** [comment]
@gabsie the problem you are seeing above is because the projects have the exact same name and the filter is filtering on facet name.  To prevent this kind of problem we will have to select by project ID not name. If you select a project that has a unique name you will only see that project selected.

**Comment by NoopDog** [comment]
@adriennes yes, we can do checkboxes instead to select the project in a row.

**Comment by NoopDog** [comment]
@adriennes this one should be closable now as well. Lets create a second ticket for the issue with projects with identical names as that is a longer term fix.

**Comment by NoopDog** [comment]
implemented
